test/sequential/test-fs-watch.js

```diff
@@ -112,12 +112,15 @@ tmpdir.refresh();
 // https://github.com/joyent/node/issues/6690
 {
   let oldhandle;
-  assert.throws(function() {
+  common.expectsError(() => {
```

Comment:
This change is actually not necessary. `assert.throws` does accept objects as well.